dask_cuda/benchmarks/utils.py

```diff
@@ -206,30 +206,21 @@ def get_cluster_options(args):
         cluster_args = [args.hosts.split(",")]
         scheduler_addr = args.protocol + "://" + cluster_args[0][0] + ":8786"
 
-        worker_options = {}
-
-        # This looks counterintuitive but adding the variable name with
-        # an empty string is how we can enable CLI booleans currently,
-        # note that SSHCluster uses the dask-cuda-worker CLI.
-        if args.enable_tcp_over_ucx:
-            worker_options["enable_tcp_over_ucx"] = ""
-        if args.enable_nvlink:
-            worker_options["enable_nvlink"] = ""
-        if args.enable_infiniband:
-            worker_options["enable_infiniband"] = ""
-        if args.enable_rdmacm:
-            worker_options["enable_rdmacm"] = ""
-
-        if args.device_memory_limit:
-            worker_options["device_memory_limit"] = args.device_memory_limit
-        if args.ucx_net_devices:
-            worker_options["ucx_net_devices"] = args.ucx_net_devices
-
         cluster_kwargs = {
             "connect_options": {"known_hosts": None},
-            "scheduler_options": {"protocol": args.protocol},
-            "worker_module": "dask_cuda.dask_cuda_worker",
-            "worker_options": worker_options,
+            "scheduler_options": {"protocol": args.protocol, "port": 8786},
```

Comment:
Just so I understand better - why do we want to explicitly set the scheduler port to 8786?

Reply:
That makes sense - thanks for the clarification!